providers/io/engrxiv/normalizer.py

```diff
@@ -0,0 +1,116 @@
+from share.normalize import ctx
+from share.normalize.parsers import Parser
+from share.normalize import tools
+
+
+class Identifier(Parser):
+    url = ctx
+    base_url = tools.Static('https://osf.io/')
```

Comment:
Making this dynamic will make it easier to test on staging https://github.com/CenterForOpenScience/SHARE/blob/develop/providers/io/osf/preprints/normalizer.py#L12